### ISSUE TITLE
Cannot resolve keyword 'storage_filename' into field

### ISSUE THREAD

**Issue by kemics** [HumanSignal/label-studio#3076]
**Describe the bug**

When I try to add `Storage filename` filter it throws exception

```
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/django/db/models/sql/query.py", line 1955, in add_fields
    join_info = self.setup_joins(name.split(LOOKUP_SEP), opts, alias, allow_many=allow_m2m)
  File "/usr/local/lib/python3.10/dist-packages/django/db/models/sql/query.py", line 1648, in setup_joins
    path, final_field, targets, rest = self.names_to_path(
  File "/usr/local/lib/python3.10/dist-packages/django/db/models/sql/query.py", line 1562, in names_to_path
    raise FieldError("Cannot resolve keyword '%s' into field. "
django.core.exceptions.FieldError: Cannot resolve keyword 'storage_filename' into field. Choices are: annotations, cancelled_annotations, comment_authors, comment_count, created_at, data, drafts, file_upload, file_upload_id, id, inner_id, io_storages_azureblobimportstoragelink, io_storages_gcsimportstoragelink, io_storages_localfilesimportstoragelink, io_storages_redisimportstoragelink, io_storages_s3importstoragelink, is_labeled, last_comment_updated_at, locks, meta, overlap, predictions, project, project_id, total_annotations, total_predictions, unresolved_comment_count, updated_at, updated_by, updated_by_id

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/usr/local/lib/python3.10/dist-packages/django/utils/decorators.py", line 43, in _wrapper
    return bound_method(*args, **kwargs)
  File "/label-studio/label_studio/data_manager/api.py", line 202, in get
    queryset = self.get_task_queryset(request, prepare_params)
  File "/label-studio/label_studio/data_manager/api.py", line 173, in get_task_queryset
    return Task.prepared.only_filtered(prepare_params=prepare_params)
  File "/label-studio/label_studio/data_manager/managers.py", line 630, in only_filtered
    return queryset.prepared(prepare_params=prepare_params)
  File "/label-studio/label_studio/data_manager/managers.py", line 451, in prepared
    queryset = apply_filters(queryset, prepare_params.filters, project, request)
  File "/label-studio/label_studio/data_manager/managers.py", line 326, in apply_filters
    value_type = type(queryset.values_list(field_name, flat=True)[0]).__name__
  File "/usr/local/lib/python3.10/dist-packages/django/db/models/query.py", line 867, in values_list
    clone = self._values(*_fields, **expressions)
  File "/usr/local/lib/python3.10/dist-packages/django/db/models/query.py", line 835, in _values
    clone.query.set_values(fields)
  File "/usr/local/lib/python3.10/dist-packages/django/db/models/sql/query.py", line 2272, in set_values
    self.add_fields(field_names, True)
  File "/usr/local/lib/python3.10/dist-packages/django/db/models/sql/query.py", line 1982, in add_fields
    raise FieldError("Cannot resolve keyword %r into field. "
django.core.exceptions.FieldError: Cannot resolve keyword 'storage_filename' into field. Choices are: annotations, cancelled_annotations, comment_authors, comment_count, created_at, data, drafts, file_upload, file_upload_id, id, inner_id, io_storages_azureblobimportstoragelink, io_storages_gcsimportstoragelink, io_storages_localfilesimportstoragelink, io_storages_redisimportstoragelink, io_storages_s3importstoragelink, is_labeled, last_comment_updated_at, locks, meta, overlap, predictions, project, project_id, total_annotations, total_predictions, unresolved_comment_count, updated_at, updated_by, updated_by_id
```

**To Reproduce**

Set such filter with any value:

<img width="515" alt="Screenshot 2022-10-11 at 18 52 15" src="https://user-images.githubusercontent.com/14817537/195125176-7435b068-2a50-402b-bec0-46ae47713e67.png">

**Environment (please complete the following information):**
 - Label Studio Version 1.6

**Comment by makseq** [comment]
Thank you for your feedback, we have created ticket about this, hope we will fix it soon [LSDV-4876].

**Comment by Gunj23** [comment]
@makseq Please fix this issue. Or is there any way to access a particular file on labelling interface using file name, when files are synced through s3 bucket (image labelling in my case)?

**Comment by makseq** [comment]
you can read task data and extract path to your storage file from it, e.g.: 
1. click `Show task source` button on the data manager page on the task row in the right. 
2. check how task data looks like, it should contain presigned URL with your media source. You will see that this URL looks very similar to the path for your bucket file.

**Comment by jorgejgnz** [comment]
The problem is still present in 1.8.0 when you try sorting by storage filename while having data synchronized with an S3 bucket.
Is there any way to fix it?

**Comment by hogepodge** [comment]
Reported in community Slack, will re-raise issue to the team.

**Comment by jombooth** [comment]
Hi @kemics, @jorgejgnz, and @Gunj23 - this was fixed in our latest release, [1.11.0](https://github.com/HumanSignal/label-studio/releases/tag/1.11.0)!
